packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -360,7 +364,7 @@ export const CallAgentProvider: (props: CallAgentProviderProps) => JSX.Element;
 // @public
 export interface CallAgentProviderProps {
     // (undocumented)
-    callAgent?: CallAgent;
+    callAgent?: CallAgent | /* @conditional-compile-remove(teams-identity-support) */ TeamsCallAgent;
```

Comment:
I noticed that we have some cc flags in our stable api md, is this expected?
There are some other cc flags from other features in stable api md as well.

Reply:
Yep, we will keep it until we remove the comment flag - maybe that would be an improvement item to cc system!